src/pages/Main/Manual/Manual.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import styles from '../MainPage.module.scss';
+import iconManual001 from '../../../assets/icons/icon_manual_001.png';
+import iconManual002 from '../../../assets/icons/icon_manual_002.png';
+import iconManual003 from '../../../assets/icons/icon_manual_003.png';
+import iconManual004 from '../../../assets/icons/icon_manual_004.png';
```

Comment:
Can we add normal names for icons? We should know what is this without open the icon. For example, 'UserAvatarIcon' or 'FollowIcon'.

Reply:
fix